WMCDuplicateRemover.Tests/WMCDuplicateRemover.Tests.csproj

```diff
@@ -37,11 +39,20 @@
     <AssemblyOriginatorKeyFile>WMCDuplicateRemoverKey.pfx</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Castle.Core, Version=3.3.0.0, Culture=neutral, PublicKeyToken=407dd0808d44fbdc, processorArchitecture=MSIL">
+      <HintPath>..\packages\Castle.Core.3.3.3\lib\net45\Castle.Core.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
     <Reference Include="ehRecObj">
       <HintPath>..\..\..\..\..\..\..\Windows\ehome\ehRecObj.dll</HintPath>
     </Reference>
-    <Reference Include="nunit.framework">
```

Comment:
And that way, you don't have to rely on anyone else who tries to execute the code to have nunit installed themselves?